File: Desarrollo de Sistemas Distribuidos/Practica7/ejercicio5.cpp
/*
	IPN - ESCOM
	Desarrollo de Sistemas Distribuidos
	4CM1

	Equipo 5
	=> Hernández Jiménez Enrique
	=> ww ww ww
	=> Fernandez Eduardo

	ww.com/sistemasdistribuidos
*/

#include <iostream>
#include <cstdlib>
#include <cmath>
using namespace std;

int main(int argc, char **argv)
{
	double i = 0, seno, coseno, tangente, logaritmo, raiz, max = atof(argv[1]);

	while(i < max)
	{
		seno += sin(i);
		coseno += cos(i);
		tangente += tan(i);
		logaritmo += log10(i);
		raiz += sqrt(i);
		i++;
	}

	return 0;
}

File: Desarrollo de Sistemas Distribuidos/Practica7/Ejercicio5.java
/*
	IPN - ESCOM
	Desarrollo de Sistemas Distribuidos
	4CM1

	Equipo 5
	=> Hernández Jiménez Enrique
	=> ww ww ww
	=> Fernandez Eduardo

	ww.com/sistemasdistribuidos
*/

//package com.ww.sistemasdistribuidos;

import java.math.*;

public class Ejercicio5
{
	public static void main(String args[])
	{
		double i = 0, seno = 0, coseno = 0, tangente = 0, logaritmo = 0, raiz = 0, max = Double.parseDouble(args[0]);
		
		while(i < max)
		{
			seno += Math.sin(i);
			coseno += Math.cos(i);
			tangente += Math.tan(i);
			logaritmo += Math.log10(i);
			raiz += Math.sqrt(i);
			i++;
		}
	}
}